x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -6,32 +6,37 @@ within your {es} cluster. Connections are secured using Transport Layer Security
 (TLS), which is commonly referred to as "SSL".
 
 WARNING: Clusters that do not have encryption enabled send all data in plain text
-including passwords. If the {es} {security-features} are enabled, unless you have a trial license, you must configure SSL/TLS for internode-communication.
-
-The following steps describe how to enable encryption across the various
-components of the {stack}. You must perform each of the steps that are
-applicable to your cluster.
+including passwords. If the {es} {security-features} are enabled, unless you
+have a trial license, you must configure SSL/TLS for internode-communication.
 
 . Generate a private key and X.509 certificate for each of your {es} nodes. See
 <<node-certificates>>.
 
 . Configure each node in the cluster to identify itself using its signed
-certificate and enable TLS on the transport layer. You can also optionally
-enable TLS on the HTTP layer. See
-<<tls-transport>> and
-<<tls-http>>.
+certificate and enable TLS on the transport layer. See <<tls-transport>>. 
+
+. Optionally enable TLS on the HTTP layer. See <<tls-http>>.
++
+--
+NOTE: To enable {kib}, {ls}, and Beats to communicate securely with {es}, you
+must configure TLS for HTTP. Likewise, it enables your users to communicate
+securely with {es} via their browsers.
+
+--
+
+Now you can proceed with the configurations that are applicable to your cluster:
 
-. Configure the {monitor-features} to use encrypted connections. See <<secure-monitoring>>.
+* Configure the {monitor-features} to use encrypted connections. See <<secure-monitoring>>.
```

Comment:
Can these be performed in any order? If not, the ordered steps might make more sense.

Reply:
I would be tempted to say no, but I'll leave to @lcawl to confirm.